src/content/add-to-app/ios/project-setup.md

```diff
@@ -5,75 +5,71 @@ description: Learn how to integrate a Flutter module into your existing iOS proj
-
-1. **Use the CocoaPods dependency manager and installed Flutter SDK.**
-  In this case, the `flutter_module` is compiled from
-  the source each time the app is built. (Recommended.)
-
-1. **Create frameworks for the Flutter engine, your compiled Dart code,
-  and all Flutter plugins.** Here, you manually embed the frameworks,
-  and update your existing application's build settings in Xcode.
-  This can be useful for teams that don't want to require every developer
-  to have the Flutter SDK and Cocoapods installed locally.
-
-1. **Create frameworks for your compiled Dart code,
-  and all Flutter plugins. Use CocoaPods for the Flutter engine.** 
-  With this option, embed the frameworks for your application
-  and the plugins in Xcode, but distribute the
-  Flutter engine as a CocoaPods podspec.
-  This is similar to the second option, but it provides
-  an alternative to distributing the large Flutter.xcframework.
-
-For examples using an app built with UIKit, 
-see the iOS directories in the [add_to_app code samples][]. 
-For an example using SwiftUI, see the iOS directory in [News Feed App][].
+application as embedded frameworks.
+To embed Flutter in your existing application,
+consider one of the following three methods.
+
+**Use the CocoaPods dependency manager and installed Flutter SDK.**
+: Each time you build the app,
+  it compiles the `flutter_module` from source.
+  (Recommended.)
+
+**Create frameworks for each component.**
+: Create frameworks for the Flutter engine, your compiled Dart code,
+  and all Flutter plugins in Xcode.
+  Embed these frameworks and update your existing application's build settings.
+  This method works for teams that don't want to require every developer
+  to install the Flutter SDK and Cocoapods on their local machines.
+
+**Create frameworks for Dart code and Flutter plugins. Use CocoaPods for the Flutter engine.**
+: Embed the frameworks for your iOS app and the plugins in Xcode,
+  but distribute the Flutter engine as a CocoaPods podspec.
+  This works like the second method,
+  but it provides an alternative to distributing the large Flutter.xcframework.
+
+For examples using an app built with UIKit,
+see the iOS directories in the [add_to_app code samples][].
+For an example using SwiftUI, consult the iOS directory in [News Feed App][].
+
+## Watch the video
+
+To learn more, watch this short video on adding Flutter to an iOS app:
+
+<iframe class="full-width" src="{{site.yt.embed}}/IIcrfrTshTs" title="Learn about how to add Flutter to an existing iOS app" {{site.yt.set}}></iframe>
 
 ## System requirements
 
 Your development environment must meet the
-[macOS system requirements for Flutter][]
-with [Xcode installed][].
-Flutter supports iOS 12 and later.
-Additionally, you will need [CocoaPods][]
-version 1.10 or later.
-
+[macOS system requirements for Flutter][] with [Xcode installed][].
+Flutter supports iOS {{site.appmin.xcode}} and later and [CocoaPods][] {{site.appmin.cocoapods}} or later.
```

Comment:
```suggestion
Flutter supports iOS {{site.appmin.ios}} and later and [CocoaPods][] {{site.appmin.cocoapods}} or later.
```

But why is this 17?   Flutter supports iOS 12.
https://github.com/flutter/website/blob/b6af70700133b17968a7a4484455b300f9111c6b/src/_data/site.yml#L77

Reply:
Oh, I see this should be `site.targetmin.ios`?

Reply:
This one wasn't fixed.
<img width="538" alt="Screenshot 2024-06-13 at 4 56 01 PM" src="https://github.com/flutter/website/assets/682784/b33c742f-2afb-499f-a5b0-24bf9f1eb047">

Reply:
Bigger issue was that this should have said Xcode 14 or later, not iOS. 🫨